docs/reference/query-languages/esql/kibana/docs/functions/abs.md

```diff
@@ -5,7 +5,7 @@ This is generated by ESQL’s AbstractFunctionTestCase. Do no edit it. See ../RE
 ### ABS
 Returns the absolute value.
 
-```
+```esql
```

Comment:
I have a script that already adds it on the kibana side. If you are going to add it always I can remove it from the kibana implementation but if this is something that wil get missed is better to remove it from here and let kibana handle it. Is it added manually or by a script?

Reply:
Added by script. It turned out that the two places I added the back-ticks were not identical. We need it for the normal docs, but missed it for the Kibana docs. Considering how I'd like to try move the two places closer together (reuse more code), it is better if I keep adding it.

Reply:
ok makes sense, I will remove it from our script then